socorro/external/boto/crash_data.py

```diff
@@ -79,3 +83,34 @@ def get(self, **kwargs):
             # contains information about buckets and prefix keys.
             # Re-wrap it here so the message is just the crash ID.
             raise CrashIDNotFound(params.uuid)
+
+
+class TelemetryCrashData(TelemetryBotoS3CrashStorage):
+    """Fetches data from TelemetryBotoS3CrashStorage"""
+
+    def __init__(self, *args, **kwargs):
+        super(TelemetryCrashData, self).__init__(*args, **kwargs)
+        # Forcibly set this to override the default in the base crash storage
+        # class for boto. We're confident that at this leaf point we want to
+        # NOT return a DotDict but just a plain python dict.
+        self.config.json_object_hook = dict
+
+    def get(self, **kwargs):
+        """Return JSON data of a crash report, given its uuid."""
+        filters = [
+            ('uuid', None, str),
+        ]
+        params = external_common.parse_arguments(filters, kwargs, modern=True)
+
+        if not params.uuid:
+            raise MissingArgumentError('uuid')
+
+        try:
+            return self.get_unredacted_processed(params.uuid)
+        except CrashIDNotFound as cidnf:
+            self.config.logger.error('telemetry crash not found: %s' % cidnf)
+            # The CrashIDNotFound exception that happens inside the
+            # crashstorage is too revealing as exception message contains
+            # information about buckets and prefix keys. Re-wrap it here so the
+            # message is just the crash ID.
+            raise CrashIDNotFound(params.uuid)
```

Comment:
This pretty much mirrors the `SimplifiedCrashData` one. We don't really need the data from the telemetry bucket right now. Mostly we just need to know if the crash is in there. So it's possible this is overkill.